rental.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+#  create class Rental
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Similarly, `book` and `person` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ to it. Kindly ensure to improve upon this aspect.